src/panel/poi/PoiPanel.jsx

```diff
@@ -202,94 +192,57 @@ export default class PoiPanel extends React.Component {
-        </div>
-      </div>
-      <div
-        className={classnames('poi_panel', { 'poi_panel--card': this.state.card })}
-        ref={this.cardRef}
-      >
-        <div className="poi_panel__content__card">
-          { isFromCategory &&
-            <div className="poi_panel__close" onClick={this.backToList}>
-              <i className="icon-x" />
-            </div>
-          }
-          <div className="poi_panel__description_container">
-            <div>
-              <PoiHeader poi={poi} />
-              <OpeningHour poi={poi} />
-            </div>
-            <PoiTitleImage poi={poi} iconOnly={true} />
-          </div>
-          <div className="poi_panel__content__card__action__container">
-            { this.isDirectionActive &&
-              <button
-                className={
-                  'poi_panel__content__card__action poi_panel__content__card__action__direction'
-                }
-                onClick={this.openDirection}
-              >
-                <span className="icon-corner-up-right" />{' '}
-                { _('DIRECTIONS', 'poi panel') }
-              </button>
-            }
-            <button className="poi_panel__content__card__action" onClick={this.showDetail}>
-              <span className="icon-chevrons-right" />{' '}
-              { _('SEE MORE', 'poi panel') }
-            </button>
-          </div>
+
+    const header = <div className="poi_panel__header">
+      {backAction &&
+        <div className={backAction.className} onClick={backAction.callback}>
+          <i className="poi_panel__back icon-arrow-left" />
+          <span className="poi_panel__back_text">{backAction.text}</span>
+          {pagesjaunes}
         </div>
-        <div className="poi_panel__content">
-          <div className="poi_panel__container">
-            <div className="poi_panel__description_container" onClick={this.center}>
-              <PoiHeader poi={poi} />
-              <PoiTitleImage poi={poi} iconOnly={false} />
-            </div>
-            <ActionButtons
-              poi={poi}
-              isFromCategory={isFromCategory}
-              isFromFavorite={isFromFavorite}
-              isDirectionActive={this.isDirectionActive}
-              openDirection={this.openDirection}
-              openShare={this.openShare}
-              isMasqEnabled={this.isMasqEnabled}
-            />
-            <PoiBlockContainer poi={poi} />
-            {poi.id.match(/latlon:/) && <div className="service_panel__categories--poi">
-              <h3 className="service_panel__categories_title">
-                <span className="icon-icon_compass" />{_('Search around this place', 'poi')}
-              </h3>
-              <CategoryList />
-            </div>}
-            {isFromOSM(poi) && <OsmContribution poi={poi} />}
-          </div>
+      }
+      {!backAction && pagesjaunes &&
+        <div className="poi_panel__pj_logo_container">{pagesjaunes}</div>
+      }
```

Comment:
This block should be rendered when accessing "pagesjaunes" POIs via their URL, but `backAction` is always defined...

Reply:
Thanks for your vigilance :) :policeman: 
I think I fixed this behavior.